Upstream-Status: Pending

Remove duplicate file entries as this breaks install with new Automake

JL 19/03/10

Index: claws-mail-2.9.1/src/gtk/Makefile.am
===================================================================
--- claws-mail-2.9.1.orig/src/gtk/Makefile.am	2007-03-22 07:52:47.000000000 +0000
+++ claws-mail-2.9.1/src/gtk/Makefile.am	2010-03-19 14:54:06.565828281 +0000
@@ -44,7 +44,6 @@
 	filesel.h \
 	foldersort.h \
 	gtkaspell.h \
-	gtksctree.h \
 	gtkshruler.h \
 	gtksctree.h \
 	gtksourceprintjob.h \
@@ -57,7 +56,6 @@
 	menu.h \
 	pluginwindow.h \
 	prefswindow.h \
-	gtkvscrollbutton.h \
 	progressdialog.h \
 	quicksearch.h \
 	sslcertwindow.h \
